statsmodels/tsa/base/tsa_model.py

```diff
@@ -48,8 +48,11 @@ def _init_dates(self, dates, freq):
         if dates is not None:
             if (not datetools._is_datetime_index(dates) and
                     isinstance(self.data, data.PandasData)):
-                raise ValueError("Given a pandas object and the index does "
-                                 "not contain dates")
+                try:
+                    dates = to_datetime(dates)
+                except:
```

Comment:
do you have a specific exception that can be put in here?
Blanket exceptions can hide too many problems